vinigrette.py

```diff
@@ -0,0 +1,9 @@
+import random
+import datetime
+begin = datetime.date(1912, 6, 23)
```

Comment:
Also here - please write your code inside functions and add documantation.
try using mor unformative names for your variables (if needed, you can use side notes to explain).